src/main/java/org/utplsql/cli/RunCommand.java

```diff
@@ -143,15 +177,23 @@ public int run() throws Exception {
         // Run tests.
         executorService.submit(() -> {
             try (Connection conn = ci.getConnection()) {
-                new TestRunner()
+                TestRunner testRunner = new TestRunner()
                         .addPathList(testPaths)
                         .addReporterList(reporterList)
                         .sourceMappingOptions(sourceMappingOptions[0])
                         .testMappingOptions(testMappingOptions[0])
                         .colorConsole(this.colorConsole)
                         .failOnErrors(true)
-                        .skipCompatibilityCheck(skipCompatibilityCheck)
-                        .run(conn);
+                        .skipCompatibilityCheck(skipCompatibilityCheck);
+
+                for (String includeObject: finalIncludeObjectsList){
```

Comment:
This should be done in Fluent interface manner to stay within the same pattern. Will need a change in Java API. I will try to do this soon.

Reply:
Thanks for your efforts, though! Very appreciated!

Reply:
Do you mean create a function like "includeObjects" or something like that? I also can do that, in general, if you are busy now.

Reply:
Yes, creating a function in the TestRunner class, return the TestRunner instance so we can continue to use fluent interface pattern. The need to create a loop outside just to add objects feels uncomfortable to me :)